Libs/MRML/Logic/vtkMRMLSliceLogic.cxx

```diff
@@ -549,11 +549,23 @@ void vtkMRMLSliceLogic::ProcessMRMLLogicsEvents()
     plane->SetOrigin(outPt3);
 
```

Comment:
I would update dims instead of adjusting inPt - to make it clear that we are just forcing the image dimensions to be non-zero.

<pre>
if (dims[0] < 1)
{ 
  dims[0] = 1;
}
if (dims[1] < 1)
{ 
  dims[1] = 1;
}
</pre>